cpp/include/options.h

```diff
@@ -11,44 +11,17 @@
 #ifndef CPP_INCLUDE_OPTIONS_H_
 #define CPP_INCLUDE_OPTIONS_H_
 
-#include "jniHelper.h"
-
+#include <wire.h>
```

Comment:
for header files in this module, use quote instead of bracket, like `#include "wire.h"`